CollapseLauncher/XAMLs/MainApp/MainPage.xaml.cs

```diff
@@ -641,11 +643,24 @@ private void SpawnAppUpdatedNotification()
             try
             {
                 string UpdateNotifFile = Path.Combine(AppDataFolder, "_NewVer");
+                string NeedInnoUpdateFile = Path.Combine(AppDataFolder, "_NeedInnoLogUpdate");
                 TypedEventHandler<InfoBar, object> ClickClose = new TypedEventHandler<InfoBar, object>((sender, args) =>
                 {
                     File.Delete(UpdateNotifFile);
                 });
 
+                // If the update was handled by squirrel and it needs Inno Setup Log file to get updated, then do the routine
+                if (File.Exists(NeedInnoUpdateFile))
+                {
+                    try
+                    {
+                        string InnoLogPath = Path.Combine(Path.GetDirectoryName(AppFolder), "unins000.dat");
+                        if (File.Exists(InnoLogPath)) InnoSetupLogUpdate.UpdateInnoSetupLog(InnoLogPath);
+                        File.Delete(NeedInnoUpdateFile);
+                    }
+                    catch { }
```

Comment:
Might want to do something in the catch to log the error in the event that it happens.